knowledge_base/dashboard_nyc_taxi/README.md

```diff
@@ -0,0 +1,50 @@
+# Dashboard for NYC Taxi Trip Analysis
+
+This example demonstrates how to model an AI/BI dashboard in a Databricks Asset Bundle.
+
+It includes and deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace.
+
+For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/en/dashboards/index.html#dashboards).
+
+## Prerequisites
+
+* Databricks CLI v0.232.0 or above
+
+## Usage
```

Comment:
Should the steps in here be numbered?

Reply:
We could but then should apply it in the other examples in the knowledge base as well.

What do you typically convey with numbering over plain enumeration?